.vib/spring-cloud-dataflow/goss/goss.yaml

```diff
@@ -0,0 +1,23 @@
+file:
+  /opt/bitnami/spring-cloud-dataflow/conf/application.yml:
+    exists: true
+    filetype: symlink
+    mode: "0777"
+    owner: root
+  /bitnami/spring-cloud-dataflow:
+    exists: true
+    filetype: directory
+    mode: "0775"
+    owner: root
+command:
+  user-id-test:
+    exec: id -u
+    exit-status: 0
+    stdout:
+      - "/[^0]+/"
```

Comment:
I checked this regex and it doesn't work for every case. E.g: 1001 (contains 0)

Reply:
OK, let me see how can I improve it.

Reply:
I've used your example to fix this test.